index.html

```diff
@@ -0,0 +1,46 @@
+<!DOCTYPE html>
+<html lang="en">
+
+<head>
+  <meta charset="UTF-8">
+  <meta name="viewport" content="width=device-width, initial-scale=1.0">
+
+  <link rel="stylesheet" href="css/styles.css">
+  <script src="js/sounds.js"></script>
+
+  <title>Focus</title>
+</head>
+
+<body>
+  <header>
+    <h1>Focus App</h1>
+  </header>
+
+  <div class="sounds">
```

Comment:
IFFF you wanted to do this w/ BEM naming conventions it could look something like this:

```suggestion
  <div class="sounds">
   <div class="sounds__item">
```